logging/zerolog/logger.go

```diff
@@ -32,9 +25,9 @@ type Logger struct {
 
 // Ref to https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/README.md#json-formats
 const (
-	TraceIDKey    = "trace_id"
-	SpanIDKey     = "span_id"
-	TraceFlagsKey = "trace_flags"
+	traceIDKey    = "trace_id"
+	spanIDKey     = "span_id"
+	traceFlagsKey = "trace_flags"
 )
 
 type ExtraKey string
```

Comment:
This ExtraKey has no use anymore.

Reply:
They are still being used in the default hook function.

Reply:
I searched, and it seems that it's no longer available. It has already been deleted.

Reply:
https://github.com/hertz-contrib/obs-opentelemetry/blame/8af103178b542475178f69100fed38f3b79b452f/logging/zerolog/option.go#L79-L107

Reply:
@rogerogers I just changed it's name

![image](https://github.com/hertz-contrib/obs-opentelemetry/assets/17726269/6bab511b-3395-4269-b84c-609fa30da8a0)

Reply:
I mean `type ExtraKey string`.

Reply:
I see. Just removed.